pkg/tenant/tenants.go

```diff
@@ -385,6 +385,10 @@ func (s *Server) TenantStats(c *gin.Context) {
 			Name:  "keys",
 			Count: int64(totalKeys),
 		},
+		{
+			Name:  "usage_kbytes",
+			Count: 0,
+		},
```

Comment:
Requested by @masskoder to include the data usage in the Tenant stats even though we have no way of populating this field yet.